ansible/group_vars/riakcs.yml

```diff
@@ -0,0 +1,7 @@
+#datavol_device=... # defined in commcare-hq/fab/inventories/* if applicable
+# http://docs.basho.com/riak/latest/ops/tuning/linux/#Storage-and-File-System-Tuning
+datavol_opts: barrier=0,data=writeback
```

Comment:
@dannyroberts this is moving from [hq prod inventory](https://github.com/dimagi/commcare-hq/blob/3508bc5446f97ecb34ccfe2a9c6a4873581fc9e0/fab/inventory/production#L61-L62) since it's a configuration detail. The only thing left in the inventory is whether or not the hosts have a `datavol_device`. I'll be submitting a PR to remove those lines from the prod inventory soon.